core/src/main/res/values/strings.xml

```diff
@@ -765,6 +765,14 @@
     <string name="rating_later_label">Remind me later</string>
     <string name="rating_now_label">Sure, let\'s do this!</string>
 
+    <!-- Share episode dialog -->
+    <string name="share_episode_label">Share…</string>
+    <string name="share_episode_positive_label_button">Share</string>
```

Comment:
Is there a string that already contains this text? If so, please use the old one instead. This reduces inconsistency when translators translate them differently.

Reply:
Yes, there was a "Share..." string. I updated and I deleted the extra I created before